packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts

```diff
@@ -23,6 +23,7 @@ export const getAudioReceiverMqa = ({audioReceiver, statsResults, lastMqaDataSen
   }
 
   audioReceiver.common.common.direction = statsResults[mediaType].direction;
+  audioReceiver.common.common.isMain = !mediaType.includes('share');
```

Comment:
can we have some unit tests for this?

Reply:
Updated, though I should note that we don't have a lot of unit tests currently to verify the MQA data, so I think someone needs to go through and add tests for all of them.